src/main/webapp/scripts/tests/comment-post-test.js

```diff
@@ -0,0 +1,77 @@
+import { postVideoComment } from '/scripts/post-comment.js';
+
+const VIDEO_ID = 'sample video id';
+const COMMENT_TEXT = 'sample comment';
+const REQUEST_BODY = {
+    'path': '/youtube/v3/commentThreads?part=snippet',
+    'method': 'POST',
+    'body': {
+        "snippet": {
+            "videoId": VIDEO_ID,
+            "topLevelComment": {
+                "snippet": {
+                    "textOriginal": COMMENT_TEXT,
+                },
+            },
+        },
+    },
+};
+
+QUnit.module('Post a comment', {
+    before: loadClientAsync,
+    beforeEach: () => { sinon.stub(gapi.client, 'request'); },
+    afterEach: () => { gapi.client.request.restore(); },
+});
+
+QUnit.test('Throw an error', function (assert) {
```

Comment:
Test names should be more descriptive. Comment post failure returns error.

Reply:
Done